code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm

```diff
@@ -44,6 +39,26 @@
 	var/mob/living/carbon/human/blessed_human = null
 	var/teleport_cooldown
 	var/teleport_cooldown_time = 20 SECONDS
+	var/swords = 0
+
+/mob/living/simple_animal/hostile/abnormality/despair_knight/ZeroQliphoth(mob/living/carbon/human/user)
+	switch(swords)
+		if(0)
+			add_overlay(mutable_appearance('ModularTegustation/Teguicons/48x48.dmi', "despair_sword1", -ABOVE_MOB_LAYER))
+			playsound(get_turf(src), 'sound/abnormalities/despairknight/attack.ogg', 50, 0, 4)
+			datum_reference.qliphoth_change(1)
+		if(1)
+			add_overlay(mutable_appearance('ModularTegustation/Teguicons/48x48.dmi', "despair_sword2", -ABOVE_MOB_LAYER))
+			playsound(get_turf(src), 'sound/abnormalities/despairknight/attack.ogg', 50, 0, 4)
+			datum_reference.qliphoth_change(1)
+		else
+			cut_overlays()
+			if(blessed_human)
+				BlessedDeath()
+			else
+				BreachEffect()
+			return
+	swords += 1
```

Comment:
Seems a bit odd; Is there any reason to *kill* the blessed? Don't turn KoD into ML copy.

Reply:
This proc does not actually kill the blessed.